packages/api/src/auth/decoders/azureActiveDirectory.ts

```diff
@@ -0,0 +1,49 @@
+import jwt from 'jsonwebtoken'
+import jwksClient from 'jwks-rsa'
+
+export const verifyAzureActiveDirectoryToken = (
+  bearerToken: string
+): Promise<null | Record<string, unknown>> => {
+  return new Promise((resolve, reject) => {
+    const { AZURE_ACTIVE_DIRECTORY_AUTHORITY } = process.env
+    if (!AZURE_ACTIVE_DIRECTORY_AUTHORITY) {
+      throw new Error(
+        '`AZURE_ACTIVE_DIRECTORY_AUTHORITY` env vars are not set.'
+      )
+    }
+
+    // @docs:   https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#sample-response
```

Comment:
JSDoc constructs, like `@docs` usually don't work in `//` comments. Change to `/** */`

Reply:
Fixed in https://github.com/jeliasson/redwood/commit/6d7044c9d1446aec3ac838e03760f97b35296976